src/components/clusters/OpenshiftVersionSelect.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { global_warning_color_100 as warningColor } from '@patternfly/react-tokens';
+import SelectField from '../ui/formik/SelectField';
+import { AlertsContext } from '../AlertsContextProvider';
+import { useOpenshiftVersions } from '../fetching/openshiftVersions';
+import { useFormikContext } from 'formik';
+import { ClusterCreateParams } from '../../api';
+
+const OpenshiftVersionSelect: React.FC = () => {
```

Comment:
Doing this in a field level component introduces several problems:
- the initial value of the form is incorrect and if we need to reset the form it won't work correctly
- In case the cluster already exists and we want to use this to find a matching version this won't work
- we're updating the form from within the field which mangles the form properties such as touched, dirty etc.

IMHO we should fetch all necessary data for the form in a wrapping component (`NewCluster`) as we already do with pullSecret and then pass the data to the form component and let Formik initialize with the values based on the data.